include/tiled_cached.h

```diff
@@ -48,8 +48,9 @@ typedef struct {
  *
  * @return The new TiledCached.
  */
-TiledCached *tiled_cached_init(MemZone *memory_pool, sprite_t *sprite, const char *map_path,
-							   Size map_size, Size tile_size);
+__attribute__((deprecated("TiledCached is no longer actively supported"))) TiledCached *
```

Comment:
I think it would be nice to say that's deprecated on the README as well.

Reply:
Agreed, I'll work on it